Resources/Audio/DeltaV/Effects/license.txt

```diff
@@ -1,3 +1,15 @@
 crack1.ogg taken from
 https://github.com/tgstation/tgstation/blob/ebc2e02e59d36214aff1eff8250b70a7970ff43d/sound/effects/wounds/crack1.ogg
 licensed under CC BY-NC-SA 3.0
+
+railgun_charge.ogg taken from
+https://pixabay.com/sound-effects/starship-rail-gun-charge-35904/
+licensed under https://pixabay.com/service/terms/ as of 1/30/2024
+
+stab_flesh.ogg modified from
+https://pixabay.com/sound-effects/knife-stab-121084/
+liscensed under https://pixabay.com/service/terms/ as of 1/30/2024
+
+suction.ogg modified from
+https://pixabay.com/sound-effects/search/suction%20cup/
+liscensed under https://pixabay.com/service/terms/ as of 1/30/2024
```

Comment:
Doesn't this fall under standalone use by distributing the content via github:

> You cannot sell or distribute the Content (either in digital or physical form) on a Standalone basis. This includes selling or distributing Content on a Standalone basis as an image, audio, video, NFT or other digital file (including through a stock media platform), as well as a print, wallpaper, poster or on merchandise or on other physical products.

Reply:
Sound effects have been replaced with ones that fall under creative commons free use licenses, there are better ways to handle copyright, but I made the changes anyways, you're welcome.